Scholars Portal Journals.js

```diff
@@ -75,18 +78,24 @@ function doWeb(doc, url) {
 }
 
 function getURI(url){
-	if (url.indexOf("xml") != -1){
+	if (url.indexOf("/details/") != -1){
 		var a = url.indexOf("details");
```

Comment:
you'll want to use `let` rather than `var` for these, both as a matter of clean code and to not upset the linter.